### ISSUE TITLE
Get Layout prints with Halo2 tooling and decide which Layouter should we use in integration tests

### ISSUE THREAD

**Issue by CPerezz** [privacy-scaling-explorations/zkevm-circuits#1135]
In the call today, it has been decided to check a bit more in depth the circuit testing times that the integration tests are taking. 
2 tasks have been arised there:
1) Print the region layout with `SimpleFloorPlanner` and `V1` and check the differences.
2) Decide if (in case `V1` reduces a degree or something similar), if we should change the default one for our integration tests (would require some benchmarking).

This will help to decide what to do next with the excessive times for circuit integration tests.

**Comment by ed255** [comment]
I plan to work on task 1 for the SuperCircuit

**Comment by han0110** [comment]
>  I imagine there should be a chance to stack regions (that use different columns) vertically by "merging" columns

I think currently none of them could merge columns, what they could do is to fill the configured constraint system as much as possible (as less height as possible). To roughly see the difference between `SimpleFloorPlanner` and `V1`, the former could only do things like tetris, but the latter could know every tetromino beforehand and reorder them (because it run 2 pass of synthesize).

> After resolving the V1 planner issues I got a plot. And... it's the same as the SimpleFloorPlanner. So maybe we're doing something wrong?

I think this makes sense, since we usually have giant region so there is no much space for `V1` to reorganize regions to save rows.

**Comment by CPerezz** [comment]
> I think this makes sense, since we usually have giant region so there is no much space for V1 to reorganize regions to save rows.

I agree with @han0110  here. Since we have 4 CellManager/CB-based circuits. Therefore, all these are simply a giant square/rectangle containing a bunch of cells. It kinda makes sense that the V1 does no move stuff arround.

**Comment by CPerezz** [comment]
>  (TODO: implement a better solution by moving fields to the heap with Box)
We need to file an issue explaining better the error and work on it. (We can maybe add it to the `tech-debt` milestone).
